codalab/lib/bundle_util.py

```diff
@@ -354,3 +354,63 @@ def check_bundle_freezable(bundle):
             'Cannot freeze bundle %s(%s), bundle is not in a final state.'
             % (bundle.uuid, bundle.metadata.to_dict()["name"])
         )
+
+
+def get_bundle_state_details(bundle):
+    """
+    This helper returns information about what the given bundle's state means.
```

Comment:
To be technically correct, should mention that it's not just the `state`, but also `run_status` or `staged_status` that constitute the string.  Maybe we should call it `get_bundle_status_description` rather than `state_details` to avoid confusion?

Reply:
I feel that `status_description` would cause more confusion than `state_details`. This data is always going to be coupled with `state` data from a UI perspective, so `state` and `state_details` seems more natural imo.